internal/testdata/addons-indeximage/reference-addon/metadata/stage/addon.yaml

```
@@ -23,4 +22,4 @@ defaultChannel: alpha
 bundleParameters: {}
 namespaceLabels: {}
 namespaceAnnotations: {}
-indexImage: quay.io/osd-addons/reference-addon-index@sha256:9b3d92cc11cd837a8e1621798ecc5de49ff05b4c437b71f6e6cb47b6cb2b384f
+indexImage: quay.io/osd-addons/reference-addon-index@sha256:b9e87a598e7fd6afb4bfedb31e4098435c2105cc8ebe33231c341e515ba9054d
```

```suggestion
indexImage: quay.io/osd-addons/reference-addon-index@sha256:d9f95ecd3cace47d9f34f63354c06d829abd165de90ef3990b6f9feda36233f3
```
Minor nitpick: preferring for the latest image :)